polaris-react/src/components/Box/Box.scss

```
@@ -0,0 +1,172 @@
+.root {
+  --pc-box-margin: initial;
+  --pc-box-margin-bottom: initial;
+  --pc-box-margin-left: initial;
+  --pc-box-margin-right: initial;
+  --pc-box-margin-top: initial;
+  --pc-box-padding: initial;
+  --pc-box-padding-bottom: initial;
+  --pc-box-padding-left: initial;
+  --pc-box-padding-right: initial;
+  --pc-box-padding-top: initial;
+
+  display: block;
+  margin: var(--pc-box-margin);
```

We could remove the above lines of code by saying if the custom property isn't defined then it falls back to `initial`.

```suggestion
  margin: var(--pc-box-margin, initial);
```